components/cpm/Analysis.vue

```diff
@@ -2,50 +2,107 @@
   <div class="w-full h-full">
     <span class="space-x-3 p-3">
       <CpmDropdown v-model="selectedVariable" :options="variables" alttext="Choose a variable." />
-      <CpmDropdown v-model="selectedResolution" :options="resolution" alttext="Select a resolution." />
-      <CpmDropdown v-model="selectedCategory" :options="categories" alttext="Choose a category" />
-      <CpmDropdown v-model="selectedDate" :options="dates" alttext="Select a date." />
+      <CpmDropdown v-model="selectedSeason" :options="seasons" alttext="Select a season." />
+      <CpmDropdown v-model="selectedModel" :options="models" alttext="Choose a model/project" />
```

Comment:
Thanks for the comment @bvreede . It sounds good. But if we put some initial text rather than the combination of actual options, then it does not make sense to put some default maps there. I would prefer the user switch to this tab and see the maps first. Then they will figure out that the dropdown menu gives them more options, and when they try to interact with the dropdown menu they will see the meaning of each variable popping up. What do you think?

The "model/group" is changed to "model/project". Thanks for finding it :stuck_out_tongue_winking_eye:.